xml/System/String.xml

```diff
@@ -14018,13 +14043,13 @@ A string is empty if it  is explicitly assigned an empty string ("") or <xref:Sy
   
  White-space characters are defined by the Unicode standard. The <xref:System.String.Trim> method removes any leading and trailing characters that produce a return value of `true` when they are passed to the <xref:System.Char.IsWhiteSpace%2A?displayProperty=nameWithType> method.  
   
-   
+[!INCLUDE[interactive-note](~/includes/csharp-interactive-note-some.md)]  
```

Comment:
this note should be moved to the remarks section of the membergroup element

Reply:
Done.